.github/workflows/pkgcheck.yml

```diff
@@ -0,0 +1,28 @@
+name: CI pkgcheck
+on: [push, pull_request]
+jobs:
+  ci-pkgcheck:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        name: [
+          Ubuntu Gcc
+        ]
+        include:
+          - name: Ubuntu Gcc
```

Comment:
Capitialize GCC like the rest of the CI workflows..

Reply:
Done.